internal/receiver/databricksreceiver/factory_test.go

```diff
@@ -72,7 +72,7 @@ func TestParseConfig(t *testing.T) {
 	rcfg := cfg.Receivers[component.NewID(typeStr)].(*Config)
 	assert.Equal(t, "my-instance", rcfg.InstanceName)
 	assert.Equal(t, "abc123", rcfg.Token)
-	assert.Equal(t, "https://my.databricks.instance", rcfg.Endpoint)
+	assert.Equal(t, "https://adb-12345.6.azuredatabricks.net", rcfg.Endpoint)
```

Comment:
probably not meant to be there?

Reply:
This is a draft PR, mostly for the SEs I'm working with. Can we wait on providing feedback until this PR is ready for review?

Reply:
Ah ok! Sorry about that. Removing.